### ISSUE TITLE
jsdoc3 3.6.9 build failure

### ISSUE THREAD

**Issue by chenrui333** [jsdoc/jsdoc#1971]
👋 trying to build the latest release, but run into some build issue. The error log is as below:

<details>
<summary>build error</summary>

```
  npm ERR! code 127
  npm ERR! path /opt/homebrew/Cellar/jsdoc3/3.6.9/libexec/lib/node_modules/jsdoc
  npm ERR! command failed
  npm ERR! command sh -c npx force-resolutions
  npm ERR! npm verb cli [
  npm ERR! npm verb cli   '/opt/homebrew/Cellar/node/17.4.0/bin/node',
  npm ERR! npm verb cli   '/opt/homebrew/Cellar/node/17.4.0/libexec/lib/node_modules/npm/bin/npm-cli.js',
  npm ERR! npm verb cli   'exec',
  npm ERR! npm verb cli   '--',
  npm ERR! npm verb cli   'force-resolutions'
  npm ERR! npm verb cli ]
  npm ERR! npm info using npm@8.3.1
  npm ERR! npm info using node@v17.4.0
  npm ERR! npm timing npm:load:whichnode Completed in 0ms
  npm ERR! npm timing config:load:defaults Completed in 1ms
  npm ERR! npm timing config:load:file:/opt/homebrew/Cellar/node/17.4.0/libexec/lib/node_modules/npm/npmrc Completed in 0ms
  npm ERR! npm timing config:load:builtin Completed in 0ms
  npm ERR! npm timing config:load:cli Completed in 1ms
  npm ERR! npm timing config:load:env Completed in 0ms
  npm ERR! npm timing config:load:project Completed in 1ms
  npm ERR! npm timing config:load:file:/private/tmp/jsdoc3-20220122-47867-14ya04d/package/.brew_home/.npmrc Completed in 0ms
  npm ERR! npm timing config:load:user Completed in 0ms
  npm ERR! npm timing config:load:file:/opt/homebrew/Cellar/jsdoc3/3.6.9/libexec/etc/npmrc Completed in 1ms
  npm ERR! npm timing config:load:global Completed in 1ms
  npm ERR! npm timing config:load:validate Completed in 0ms
  npm ERR! npm timing config:load:credentials Completed in 1ms
  npm ERR! npm timing config:load:setEnvs Completed in 0ms
  npm ERR! npm timing config:load Completed in 5ms
  npm ERR! npm timing npm:load:configload Completed in 5ms
  npm ERR! npm timing npm:load:setTitle Completed in 2ms
  npm ERR! npm timing config:load:flatten Completed in 1ms
  npm ERR! npm timing npm:load:display Completed in 3ms
  npm ERR! npm verb logfile /Users/brew/Library/Caches/Homebrew/npm_cache/_logs/2022-01-23T04_41_05_603Z-debug-0.log
  npm ERR! npm timing npm:load:logFile Completed in 2ms
  npm ERR! npm timing npm:load:timers Completed in 0ms
  npm ERR! npm timing npm:load:configScope Completed in 0ms
  npm ERR! npm timing npm:load Completed in 12ms
  npm ERR! npm http fetch GET 200 https://registry.npmjs.org/force-resolutions 50ms (cache miss)
  npm ERR! npm timing arborist:ctor Completed in 1ms
  npm ERR! npm timing arborist:ctor Completed in 0ms
  npm ERR! npm WARN exec The following package was not found and will be installed: force-resolutions
  npm ERR! npm timing idealTree:init Completed in 1ms
  npm ERR! npm timing idealTree:userRequests Completed in 0ms
  npm ERR! npm sill idealTree buildDeps
  npm ERR! npm sill fetch manifest force-resolutions@*
  npm ERR! npm http fetch GET 200 https://registry.npmjs.org/force-resolutions 3ms (cache hit)
  npm ERR! npm sill placeDep ROOT force-resolutions@1.0.10 OK for:  want: *
  npm ERR! npm timing idealTree:#root Completed in 4ms
  npm ERR! npm timing idealTree:node_modules/force-resolutions Completed in 0ms
  npm ERR! npm timing idealTree:buildDeps Completed in 6ms
  npm ERR! npm timing idealTree:fixDepFlags Completed in 0ms
  npm ERR! npm timing idealTree Completed in 7ms
  npm ERR! npm timing reify:loadTrees Completed in 7ms
  npm ERR! npm timing reify:diffTrees Completed in 0ms
  npm ERR! npm sill reify moves {}
  npm ERR! npm timing reify:retireShallow Completed in 0ms
  npm ERR! npm timing reify:createSparse Completed in 0ms
  npm ERR! npm timing reify:loadBundles Completed in 0ms
  npm ERR! npm sill audit bulk request { 'force-resolutions': [ '1.0.10' ] }
  npm ERR! npm sill tarball no local data for force-resolutions@https://registry.npmjs.org/force-resolutions/-/force-resolutions-1.0.10.tgz. Extracting by manifest.
  npm ERR! npm http fetch GET 200 https://registry.npmjs.org/force-resolutions/-/force-resolutions-1.0.10.tgz 37ms (cache miss)
  npm ERR! npm timing reifyNode:node_modules/force-resolutions Completed in 39ms
  npm ERR! npm timing reify:unpack Completed in 39ms
  npm ERR! npm timing reify:unretire Completed in 0ms
  npm ERR! npm timing build:queue Completed in 0ms
  npm ERR! npm timing build:link:node_modules/force-resolutions Completed in 1ms
  npm ERR! npm timing build:link Completed in 1ms
  npm ERR! npm timing build:deps Completed in 1ms
  npm ERR! npm timing build Completed in 1ms
  npm ERR! npm timing reify:build Completed in 1ms
  npm ERR! npm timing reify:trash Completed in 0ms
  npm ERR! npm http fetch POST 200 https://registry.npmjs.org/-/npm/v1/security/advisories/bulk 131ms
  npm ERR! npm timing auditReport:getReport Completed in 131ms
  npm ERR! npm sill audit report {}
  npm ERR! npm timing auditReport:init Completed in 0ms
  npm ERR! npm timing reify:audit Completed in 131ms
  npm ERR! npm timing reify Completed in 140ms
  npm ERR! sh: force-resolutions: command not found
  npm ERR! npm timing command:exec Completed in 218ms
  npm ERR! npm verb exit 127
  npm ERR! npm timing npm Completed in 303ms
  npm ERR! npm verb code 127
```

</details>

full build log, https://github.com/Homebrew/homebrew-core/runs/4910879649
relates to Homebrew/homebrew-core#93647

**Comment by chenrui333** [comment]
I could not reproduce the issue with 3.6.7, but can reproduce the issue with 3.6.8 if that helps.

**Comment by hegemonic** [comment]
Can you try running `npx force-resolutions`? Does that work, or do you get an error?

**Comment by traverspinkerton** [comment]
Also having receiving this when building in CI. Rolling back to 3.6.7 for now until this issue is resolved

**Comment by Berrrtz** [comment]
I'm also seeing this effect Karma Tests in my CI/CD Pipeline. It references jsdoc as the root of the issue. I added jsdoc 3.6.7 to my package.json file and that fixed the issue.

![image](https://user-images.githubusercontent.com/66445766/150863284-f36a21e7-65a6-442e-933b-fdc2c3caef88.png)

**Comment by hegemonic** [comment]
I released JSDoc 3.6.10 with a fix for this issue. If one of you could confirm that it works in your CI/CD environment, I'd appreciate it.

**Comment by gpanganiban-extreme** [comment]
I can confirm that it is working correctly now on one of our Jenkins build  jobs that was previously failing with this error.
